api/lib/routes/aggregated_metrics.js

```diff
@@ -13,13 +13,13 @@ module.exports = function(settings) {
     var metricType = req.params.metric_type;
     var startTime = req.query["start-time"];
     var endTime = req.query["end-time"];
-    var order = req.query["order"];
+    var orderDirection = req.query["order-direction"];
```

Comment:
considering the backward compatibility  , could we support both "order" and "order-direction" as a valid parameter?  In the external document, we can claim the "order-direction" only.